css-three-column-layout/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <section>
+    <div class="container">
+      <h1>Designed for life science. Optimized for SaMD and Digital Health.</h1>
+      <div class="row">
+        <div class="column1">
+          <span class="circle"></span>
+        </div>
+        <div class="column1">
+          <span class="circle"></span>
+        </div>
+        <div class="column1">
+          <span class="circle"></span>
+        </div>
+        <div class="pictures">
+          <img src="images/merge.png" alt="merge" width="45" height="45">
+        </div>
+        <div class="pictures1">
+          <img src="images/cloud-computing.png" alt="cloud-computing" width="45" height="45">
+        </div>
+        <div class="pictures3">
+          <img src="images/discussion.png" alt="discussion" width="45" height="45">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column2">
+          <h2>Streamlined QMS</h2>
+        </div>
+        <div class="column2">
+          <h3>Integrates with existing tools</h3>
+        </div>
+        <div class="column2">
+          <h4>In-depth Onboarding</h4>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column3">
+          <p>A QMS that includes modules for all<br>
+            stages of the product life cycle from<br>    Design Control to CAPA</p>
+        </div>
+        <div class="column3">
+          <p>Enzyme QMS integrates with your<br>
```

Comment:
Try reorganizing this into 1 row of 3 columns, each column should have an image, heading, and paragraph in it.